proto/umee/leverage/v1beta1/query.proto

```diff
@@ -240,5 +240,5 @@ message QueryLiquidationTargetsRequest {}
 // QueryLiquidationTargetsResponse defines the response structure for the ExchangeRate
 // gRPC service handler.
 message QueryLiquidationTargetsResponse {
-  repeated string targets = 1 [(gogoproto.nullable) = false];
+  repeated string targets = 1;
```

Comment:
Note: nullable flag removed because `make proto-gen` was warning that the flag was redundant on a non-nullable native type.